src/devices/futurehome.ts

```diff
@@ -0,0 +1,51 @@
+import {Definition} from 'src/lib/types';
+import * as exposes from '../lib/exposes';
+import * as tuya from '../lib/tuya';
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const definitions: Definition[] = [
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_e5hpkc6d']),
```

Comment:
It looks this device is already supported:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/d69dc7a13fbf9254219c5cfda84e50f5629bf1d0/src/devices/connecte.ts#L10

Reply:
It seems like support was added last week when I look at the blame. I started work on this a couple months ago and back then there wasn't support for it at all. I'll try updating my zigbee2mqtt container and giving it a try assuming it's been updated with those changes yet.

That said, there is an argument to be made about approach here. The other implementation is using what is labeled as "legacy" APIs. Is that the best approach? While I haven't tested it yet, I'm also unsure what I feel about the MQTT interface of the other approach. I'll report back when I've been able to try it myself.

If I find my approach to be better in my own opinion, and you agree, I can implement what I missed based on the existing implementation and submit this as an improvement rather than adding support for something new. I'll get back with proper arguments for that if so.

Reply:
I definitely like your approach, if you could update the existing converter that would be great!

Reply:
Sounds good! I'll try to get around to it sometime this week then!

One question regarding this though. The manufacturer name. I looked into this a bit. Futurehome and Connecte actually merged back in 2022 and it seems that the devices are now sold under the Futurehome brand instead. Should I change the manufacturer name to Futurehome or keep it as Connecte like the old approach did?

Reply:
I would show the vendor name whatever is on the product/box